app/src/main/java/com/example/lawn_care/viewYourProperties.java

```diff
@@ -97,10 +106,28 @@ public void onResponse(String response) {
                                     //you can change the textview settings, for example
                                     TV_address.setTextSize(20);
                                     //IDEA: try changing some other attributes to make it look better
+                                    deleteButton.setText("Delete");
+                                    editButton.setText("Edit");
+                                    // Set click listener for delete listing function
+                                    deleteButton.setOnClickListener(new View.OnClickListener() {
+                                        @Override
+                                        public void onClick(View v) {
+                                            deleteEntry(propertyNum);
+                                        }
+                                    });
+                                    // Set click listener for edit listing function
+                                    editButton.setOnClickListener(new View.OnClickListener() {
+                                        @Override
+                                        public void onClick(View v) {
+                                            editEntry(propertyNum);
+                                        }
+                                    });
 
                                     //add the views to the current linear layout
                                     listingItem.addView(TV_address);
                                     listingItem.addView(TV_workNeeded);
+                                    linearLayout.addView(deleteButton);
+                                    linearLayout.addView(editButton);
```

Comment:
buttons added as another row in linear layout. maybe make them share a row, or put them on the side. this might not look pretty